apps/data/lib/data/opportunities.ex

```diff
@@ -5,7 +5,8 @@ defmodule Data.Opportunities do
 
   alias Data.{Opportunity, Repo}
 
-  def all, do: Repo.all(Opportunity)
+  def all(opts \\ [page_size: 25])
```

Comment:
Since we know we'll be supporting filters, sorting, and pagination should we consider making the options a map and handling defaults for keys not present?

Something like this:

```elixir
@defaults %{page_size: 25}

def all(opts \\ %{}) do
  opts = Map.merge(@defaults, opts)
  Repo.paginate(Opportunity, opts.page_size)
end
```